.travis.yml

```diff
@@ -41,6 +39,8 @@ install:
 
 script:
   - make test
+  - sudo apt-get install python-pip
+  - sudo pip install pylint
   - jdk_switcher use oraclejdk8
   - |-
     PYTHONPATH=libexec/python/ pylint libexec/python/ -r n --msg-template="{path}:{line}: [{msg_id}({symbol}), {obj}] {msg}" > pylint-report.txt
```

Comment:
Oh, that's just the bash syntax for temporarily setting an environment variable over the scope of a single line.